content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1275,6 +1275,25 @@ The kubectl command line tool uses this annotation as a legacy mechanism
 to track changes. That mechanism has been superseded by
 [Server-side apply](/docs/reference/using-api/server-side-apply/).
 
+### kubectl.kubernetes.io/restartedAt {#kubectl-k8s-io-restart-at}
+
+Type: Annotation
+
+Example: kubectl.kubernetes.io/restartedAt: "2024-06-21T17:27:41Z"
```

Comment:
_Minor non-blocking nit_: Adjust the formatting of the example block to match the style of all the other examples on this [page](https://kubernetes.io/docs/reference/labels-annotations-taints/).

```suggestion
Example: `kubectl.kubernetes.io/restartedAt: "2024-06-21T17:27:41Z"`
```